source/creating_doc/doc_checklist.rst

```diff
@@ -1,35 +1,109 @@
 Documentation Checklist
 =============================
 
-TBP:
+This page lists items you should check when you are creating new documentation. We recommend reviewing this list before you create a pull request.
 
-Build with no errors or warnings
+.. contents:: Contents
+  :local:
+  :depth: 1
 
-Review RTD build
+Your Audience
+**************
 
-Links in links.txt file
+Make sure your documentation is targeted to the right audience and organized under that audience's section of the documentation. Each topic in the documentation is for one of the following audiences:
 
-Common terms in substitutions.txt file
+* Educators
+* Course Operators
+* Site Operators
+* Developers
 
-New topics added to a TOC
+Ensure that new topics are stored in the directory for the appropriate audience, under the ``source`` directory.
 
-How-to/Reference/Concept structure adhered to
+For more information, see :ref:`Audiences`.
 
-Audience targets adhered to
+Topic Types
+************
 
-Change log entry made
+Each topic in the documentation is one of three types:
 
+* How to
 
+* Reference
 
+* Concepts
 
+Ensure that you have clearly identified the types of new topics you have created, and have stored new files in the directory for that topic type.
 
+For more information, see :ref:`Content Types`.
 
 
+New Topics in TOCs
+*******************
 
+Every new topic must be added to a table of contents. Add each new topic to the audience home page in the appropriate place.
 
 
+RST Standards
+**************
 
+Ensure that the new documentation adheres to RST standards and Open edX best practices. You should use :ref:`Documentation Templates` as much as possible, and review :ref:`New Documentation Guidelines`.
 
 
+Images
```

Comment:
Some of this might be repetative with the intro to RST that I PRed earlier this week.  I think we want to seperate the checklist from the part that's teaching them about RST, so maybe it's best to incorporate this into the writing RST reference.